cli/pcluster/cfnconfig.py

```diff
@@ -542,12 +542,14 @@ def __init_ec2_policies(self):
             iam_policies = list(map(lambda x: x.strip(), __temp__.split(",")))
             if self.parameters.get("Scheduler") == "awsbatch":
                 aws_batch_iam_policy = "arn:{0}:iam::aws:policy/AWSBatchFullAccess".format(get_partition(self.region))
-                if not (aws_batch_iam_policy in iam_policies):
+                if aws_batch_iam_policy not in iam_policies:
                     iam_policies.append(aws_batch_iam_policy)
             self.__validate_resource("EC2IAMPolicies", iam_policies)
             self.parameters["EC2IAMPolicies"] = ",".join(iam_policies)
         except configparser.NoOptionError:
-            pass
+            self.parameters["EC2IAMPolicies"] = "arn:{0}:iam::aws:policy/AWSBatchFullAccess".format(
```

Comment:
I think we need to check scheduler first

Reply:
Added